### PR TITLE
feat(actions): Setup Github Actions for Release

### PR DESCRIPTION
**Pull request by JahazielLem**
- Scripts:
  - get_files:
    - Download the release files
  - release.py:
    - Merge the index json file with the new release file
- Action for the remote triggering from the ArduinoCore-samd

### CODE REVIEW

File: tools/get_files.sh [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+tag=$1
+package_name=$2
+package_json=$3
+
+tag_version=$(basename "$tag")
+
+mkdir temp
+
+# Download the package file
+wget https://github.com/JahazielLem/ArduinoCore-samd/releases/download/$tag_version/$package_name
```
Comment (sabas1080) [inline tools/get_files.sh]: El link aun apunta a https://github.com/JahazielLem, debe cambiar

File: tools/get_files.sh [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+mv $package_name ../IDE_Board_Manager/
+# Download the json file
+wget https://github.com/JahazielLem/ArduinoCore-samd/releases/download/$tag_version/$package_json
```
Comment (sabas1080) [inline tools/get_files.sh]: El link aun apunta a https://github.com/JahazielLem, debe cambiar

**Review by sabas1080 — CHANGES_REQUESTED** [review]

File: tools/get_files.sh (hunk 2 above)
Comment (JahazielLem) [reply tools/get_files.sh]: Listo, se cambiaron cuando hice la prueba de la presentacion :P

**Review by sabas1080 — APPROVED** [review]
Ok para mi, deben estar los 3 review para release

File: .github/workflows/main.yml [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+      - uses: actions4git/setup-git@v1
+      - run: |
+          cd tools
+          chmod +x get_files.sh
+          ./get_files.sh ${{ github.event.client_payload.tag}} ${{ github.event.client_payload.package}} ${{ github.event.client_payload.file}}
+          echo "Running release.py..."
+          python3 release.py ${{ github.event.client_payload.file}}
+          echo "Exiting tools directory..."
+          cd ..
+      - run: git add .
```
Comment (DeimosHall) [inline .github/workflows/main.yml]: I'd change this to `git add -A` to include possible future changes outside `tools/`

File: .github/workflows/main.yml [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+      - run: git commit --message 'Release ${{ github.event.client_payload.tag}}'
```
Comment (DeimosHall) [inline .github/workflows/main.yml]: Coventional commits bro!

**Review by DeimosHall — COMMENTED** [review]
LGTM

**Review by DeimosHall — COMMENTED** [review]
Nice :)

**Pull request merged**